index.html

```diff
@@ -614,25 +505,22 @@ <h3>
                   <a>current settings object</a>.
                   </li>
                   <li>If |document| is not <a>fully active</a>, reject
-                  |promise| with a "{{NotAllowedError}}" {{DOMException}}, and
-                  run <a>abort the request steps</a>.
+                  |promise| with a "{{NotAllowedError}}" {{DOMException}},
+                  and abort these steps.
                   </li>
-                  <li data-link-for="">If |lock|'s |type| is {{
+                  <li data-link-for="">If |type| is {{
                   WakeLockType["screen"] }} and the {{Document}} of the
                   <a>top-level browsing context</a> is <a>hidden</a>, reject
-                  |promise| with a "{{NotAllowedError}}" {{DOMException}}, and
-                  run <a>abort the request steps</a>.
+                  |promise| with a "{{NotAllowedError}}" {{DOMException}},
+                  and abort these steps.
                   </li>
                 </ol>
               </li>
               <li>Let |success:boolean| be the result of awaiting
-                <a>acquire a wake lock</a> on |lock|.
+                <a>acquire a wake lock</a> with |promise| and |type|.
               </li>
               <li>If |success| is `false` then reject |promise| with a
```

Comment:
That is the point. You ask for a wake lock forever (we might add a developer set time limit in the future). It can then be aborted (exception) by system or via AbortController.